### PR TITLE
sqlite3: update all binaries

### PR DESCRIPTION
**Pull request by kewde**
Hi,

I help maintain the sqlite3 project, I've been using your project for my electron builds.
I've added the additional code and I don't mind keeping your upstream up to date with the new releases.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "sqlite3-offline",
+  "name": "sqlite3",
```
Comment (kewde) [inline package.json]: you'll want to undo this.

Comment (DenisCarriere) [reply package.json]: 👍

**Review by DenisCarriere — APPROVED** [review]
Awesome! Thanks for this PR

**Comment by DenisCarriere** [comment]
Published new npm release under the same version as `sqlite` v4.0.6

**Comment by kewde** [comment]
Thanks @DenisCarriere

**Pull request merged**